integration/dbt/scripts/dbt-ol

```diff
@@ -42,14 +93,44 @@ def main():
         print("OPENLINEAGE_URL is not set: stopping execution")
         sys.exit(1)
 
+    # We can get this if we have been orchestrated by an external system like airflow
+    external_parent_id = os.getenv("OPENLINEAGE_PARENT_ID")
+    external_run_metadata = None
+
+    client = OpenLineageClient.from_environment()
+
+    if external_parent_id:
+        external_namespace, parent_job_name, parent_run_id = external_parent_id.split('/')
+        external_run_metadata = ParentRunMetadata(
+            run_id=parent_run_id,
+            job_name=parent_job_name,
+            job_namespace=external_namespace
+        )
+
     processor = DbtArtifactProcessor(
         producer=PRODUCER,
         target=target,
         project_dir=project_dir,
-        profile_name=profile_name
+        profile_name=profile_name,
     )
 
+    # Always emit "wrapping event" around dbt run. This indicates start of dbt execution, since
+    # both the start and complete events for dbt models won't get emitted until end of execution.
+    parent_event = parent_run_event_start(
+        job_name=f"dbt-run-{processor.project['name']}",
+        external_run_metadata=external_run_metadata
+    )
+    parent_run_metadata = ParentRunMetadata(
```

Comment:
Rather than `external_parent_xxx` above and `parent_xxx` here, can we use `parent_...` and `dbt_run_xxx` respectively? Whenever I see `parent_` here, I assume it's the process that kicked off the dbt run, not the dbt run itself.

Reply:
Fixed.